### PR TITLE
Fix io alignment for mock-array-big

### PR DESCRIPTION
**Pull request by yupferris**
Also clean up some of the config/setup stuff to be a bit more straightforward.

This is basically #918 rebased onto latest master. #918 is being left as-is for debugging #919.

### CODE REVIEW

**Comment by yupferris** [comment]
This crashes currently, see #935.

**Comment by eder-matheus** [comment]
> This crashes currently, see #935.

I have this branch that should fix the crash: https://github.com/eder-matheus/OpenROAD/tree/ppl_groups. Could you try it?

**Comment by yupferris** [comment]
I've upgraded OpenROAD to a version which contains these fixes and verified that this PR gets past the previous ppl errors, however now I'm getting a divergence in global placement, which I'll investigate a bit.

**Comment by yupferris** [comment]
Tweaking some options works around this and makes the design a lot tighter with negligible additional runtime, so this should be good to go.

**Comment by yupferris** [comment]
@maliberty can you review when you get the chance? (I don't have rights to request reviews, hence asking this way)

**Comment by yupferris** [comment]
I don't understand the failures - it looks like there's some kind of results check that happens after building `mock-array-big` which now fails as the results have changed. Is this correct? Are there docs on how should I reproduce this locally and update?

**Comment by vijayank88** [comment]
@yupferris 
Following metrics failing.
```
[ERROR] detailedroute__route__wirelength fail test: 39452.0 <= 31191.0
```
If it is acceptable, do `make update_ok` in your local machine and push the changes to pass the CI.

**Comment by yupferris** [comment]
Done, thanks for the clarification.

File: tools/OpenROAD
Comment (maliberty) [inline tools/OpenROAD:1]: Is this necessary for this PR?

Comment (yupferris) [reply tools/OpenROAD:1]: Yes, otherwise we hit #935.

**Review by maliberty — APPROVED** [review]

**Pull request merged**